app-text/master-pdf-editor/master-pdf-editor-5.6.80.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop xdg-utils
+
+DESCRIPTION="A complete solution for viewing and editing PDF files"
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+SRC_URI="https://code-industry.net/public/${P}-qt5.x86_64.tar.gz"
+
+LICENSE="master-pdf-editor"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+RDEPEND="
+	>=dev-qt/qtcore-5.9.8:5
+	>=dev-qt/qtgui-5.9.8:5
+	>=dev-qt/qtnetwork-5.9.8:5
+	>=dev-qt/qtprintsupport-5.9.8:5
+	>=dev-qt/qtsvg-5.9.8:5
+	>=media-gfx/sane-backends-1.0
+"
```

Comment:
I am getting:
```
 * QA Notice: Unresolved soname dependencies:
 * 
 * 	/opt/master-pdf-editor/masterpdfeditor5: libnss3.so libsmime3.so
 * 
```
```
>: scanelf -n /opt/master-pdf-editor/masterpdfeditor5
 TYPE   NEEDED FILE 
ET_EXEC librt.so.1,libsane.so.1,libnss3.so,libnspr4.so,libsmime3.so,libQt5PrintSupport.so.5,libQt5Widgets.so.5,libQt5Gui.so.5,libQt5Network.so.5,libQt5Core.so.5,libpthread.so.0,libstdc++.so.6,libm.so.6,libgcc_s.so.1,libc.so.6 /opt/master-pdf-editor/masterpdfeditor5 
```
These are both provided by *dev-libs/nss*, I'll add it for you.